bundles/org.openhab.binding.enocean/src/main/java/org/openhab/binding/enocean/internal/transceiver/EnOceanTransceiver.java

```diff
@@ -309,8 +309,12 @@ private void processMessage(byte firstByte) {
                                             logger.debug("{} with RORG {} for {} payload {} received",
                                                     packet.getPacketType().name(), msg.getRORG().name(),
                                                     HexUtils.bytesToHex(msg.getSenderId()), HexUtils.bytesToHex(d));
-
-                                            informListeners(msg);
+                                            
+                                            if(msg.getRORG() != RORG.Unknown) {
+                                                informListeners(msg);
+                                            } else {
+                                                logger.info("Received unknown RORG, payload {}", HexUtils.bytesToHex(d));
```

Comment:
I think debug instead of info is appropriate.

Reply:
Changed logging level. Idee behing logger.info was that users find unsupported RORGs easier and can tell me.

Reply:
Thank makes sense. But how should a user know? Is it noted in the documentation? Fits a meaningful logger message better?

Reply:
@fruggy83 Can you comment here?

Reply:
HI @J-N-K, @cweitkamp, sorry for my delay. 

> But how should a user know? Is it noted in the documentation? 

You are right an info in the logs is not usefull without a hint in the docs to look in the log. I am currently working on the implementation of the last unsupported RORGs. So I would guess that logger.debug is ok as it is just a interim solution until I finished my current work.